### PR TITLE
Add better logging while loading databases

### PR DESCRIPTION
**Pull request by aeisenberg**
A few users have been mentioning that databases are being removed upon restarting vscode. This PR adds some extra logging to help debug this situation.

Logging will look like this:

```
Initializing database manager.
Found 1 persisted databases: file:///Users/andrew.eisenberg/codeql-home/databases/db1
Initializing database panel.
Registering database panel commands.
...many more log messages...
Loaded database db1 at URI file:///Users/andrew.eisenberg/NonEclipse/codeql-home/databases/db1.
Finished loading persisted databases.
```

## Checklist

- [n/a] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [n/a] Issues have been created for any UI or other user-facing changes made by this pull request.
- [n/a] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

**Review by angelapwen — COMMENTED** [review]
Confirmed that this works locally. I found it a bit hard to parse the log messages as some were interspersed between other command logs, like:

```
Resolving tests using CodeQL CLI: resolve tests --strict-test-discovery -v --log-to-stderr --format json /Users/angelapwen/repos/vscode-codeql-starter/ql...
Loaded database FreeCAD_FreeCAD_aa19e21 at URI file:///Users/angelapwen/Library/Application Support/Code/User/workspaceStorage/534f4aa59e92c11b0b4d495002af5e64/GitHub.vscode-codeql/FreeCAD_FreeCAD_python-srcVersion_aa19e216625cbfdab5099d8eea6b62122917a3f6-dist_codeql-bundle-linux64-20220428-2240588445/FreeCAD_FreeCAD_aa19e21.
Loaded database elastic_elasticsearch_7556157 at URI file:///Users/angelapwen/Library/Application Support/Code/User/workspaceStorage/534f4aa59e92c11b0b4d495002af5e64/GitHub.vscode-codeql/elastic_elasticsearch_java-srcVersion_755615781ab631354a008fbc49ca56eca0ba42bb-dist_codeql-bundle-linux64-20220728-2755768438/elastic_elasticsearch_7556157.
Loaded database u-boot_u-boot_d0d07ba at URI file:///Users/angelapwen/Library/Application Support/Code/User/workspaceStorage/534f4aa59e92c11b0b4d495002af5e64/GitHub.vscode-codeql/u-boot_u-boot_cpp-srcVersion_d0d07ba86afc8074d79e436b1ba4478fa0f0c1b5-dist_odasa-2019-07-25-linux64/u-boot_u-boot_d0d07ba.
Finished loading persisted databases.
CLI command succeeded.
```
Seems like it's happening because they're happening asynchronously. I wonder if we could easily delay the database logging until after the other "Resolving tests" logs?

**Comment by aeisenberg** [comment]
Thanks for confirming this. Yes, the log messages are interspliced with others for exactly the reason you mention. We might be able to do a better job of batching them, but that is a bigger task and one that will require some thought so that we don't accidentally avoid sending the messages to the log output.

Perhaps, we can start prefixing each log message with an informative identifier to help keep track of which command sent them.

**Comment by angelapwen** [comment]
> Perhaps, we can start prefixing each log message with an informative identifier to help keep track of which command sent them.

I like this — approving the PR as is for now!

**Review by angelapwen — APPROVED** [review]

**Comment by aeisenberg** [comment]
👍🏼 I will create an issue fo rthis.
